theforeman.org/pipelines/lib/rbenv.groovy

```diff
@@ -0,0 +1,31 @@
+def bundleInstall(version, gemfile=null) {
+    command = "bundle install"
+
+    if (gemfile) {
+        command = "${command} --gemfile=${gemfile}"
+    }
+
+    withRuby(version, "bundle config set path ~/.rubygems")
+    withRuby(version, command)
+}
+
+def bundleExec(version, command, gemfile=null) {
+    command = "bundle exec ${command}"
+
+    if (gemfile) {
+        command = "BUNDLE_GEMFILE=${gemfile} bundle exec ${command}"
```

Comment:
this leads to the command being `BUNDLE_GEMFILE=lol bundle exec bundle exec command`

Reply:
Double your bundle for safety.